pkgs/tools/misc/sexp/default.nix

```diff
@@ -0,0 +1,19 @@
+{ ocamlPackages }:
+
+with ocamlPackages;
+
+janePackage {
+  pname = "sexp";
+  hash = "0cqp6syc4ap2nxgg1mvwwz2pmib48kp3gigzpjwh20wr38qq0p1r";
+  propagatedBuildInputs = [
```

Comment:
Why are these propagated?

Reply:
sexp also provides a sexp_app library which is pretty useful.  It looks like it's not exposed as a `public_library` in the dune file, so that's unfortunate.  Should I switch it to use `buildInputs` instead?

Reply:
(I'm also looking into giving the `sexp_app` library a public name)

Reply:
If this is easy to do it here, you may patch the source to export the `sexp_app` lib. Then you can keep the propagated inputs but the package would then be more appropriately located with the other jane-street libraries.

Reply:
Okay, the patch turned out to be pretty straightforward.  I updated the PR for the remaining packages.